src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,28 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Injector injector = Injector.getInstance("mate");
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        DriverService driverService = (DriverService) injector.getInstance((DriverService.class));
```

Comment:
![Screenshot 2022-06-20 at 09 07 07](https://user-images.githubusercontent.com/60258373/174535781-5f660820-7e44-4287-a6ab-34dec78830ef.png)
Before you create PR with your homework, you need to go through the checklist under the task, and correct all the points described there. The mentor will not check the work until the checklist items are corrected

Reply:
Sure. I almost got away with it. :-)